<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core</class>
 <widget class="QWidget" name="Core">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBackup">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/cloud-computing.png</normaloff>:/img/cloud-computing.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Backup</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="bkpObjScroll">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>558</width>
            <height>294</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTreeWidget" name="srcViewTree">
             <column>
              <property name="text">
               <string>Source</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Destination</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Unsynced Files</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Unsynced Size</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnScan">
           <property name="text">
            <string>Scan</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/img/scan.png</normaloff>:/img/scan.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSync">
           <property name="text">
            <string>Backup</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/img/backup.png</normaloff>:/img/backup.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabConfig">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/setting.png</normaloff>:/img/setting.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Configure</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="btnConfReset">
           <property name="text">
            <string>Reset</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/img/refresh_1.png</normaloff>:/img/refresh_1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnConfSave">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lblConfPath">
         <property name="text">
          <string>Config file path: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Rules list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>558</width>
            <height>283</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="ltRulesList">
             <item>
              <widget class="QListWidget" name="lstRules"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QPushButton" name="btnRuleEdit">
                 <property name="text">
                  <string>Edit</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/img/edit.png</normaloff>:/img/edit.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnRuleDel">
                 <property name="text">
                  <string>Remove</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/img/remove.png</normaloff>:/img/remove.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnRuleAdd">
                 <property name="text">
                  <string>Add</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/img/new.png</normaloff>:/img/new.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="ltRuleEdit">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Rule Details</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leRuleName">
               <property name="placeholderText">
                <string>Rule name</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLineEdit" name="leRuleSrc">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>Source path</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnRuleSrc">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLineEdit" name="leRuleDst">
                 <property name="placeholderText">
                  <string>Destination path</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnRuleDst">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Rule Description</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="txtRuleDescription"/>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btnRuleApply">
                 <property name="text">
                  <string>Apply</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/img/commit.png</normaloff>:/img/commit.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnRuleDiscard">
                 <property name="text">
                  <string>Discard</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/img/cancel_edit.png</normaloff>:/img/cancel_edit.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
